nodejs/__test__/table.test.ts

```diff
@@ -331,7 +330,38 @@ describe.each([arrow15, arrow16, arrow17, arrow18])(
       const table = await db.createTable("my_table", data);
       expect(await table.countRows()).toEqual(2);
     });
-  },
+
+    it("should allow undefined and omitted nullable vector fields", async () => {
+      // Test for the bug: can't pass undefined or omit vector column
+      const db = await connect("memory://");
+      const schema = new arrow.Schema([
+        new arrow.Field(
+          "vector",
+          new arrow.FixedSizeList(
+            32,
+            new arrow.Field("item", new arrow.Float32(), true)
+          ),
+          true // nullable = true
+        ),
+      ]);
+      const table = await db.createEmptyTable("test_table", schema);
+
+      // Should not throw error for undefined value
+      await table.add([{ vector: undefined }]);
+
+      // Should not throw error for omitted field
+      await table.add([{}]);
+
+      // Should still work for null
+      await table.add([{ vector: null }]);
+
+      // Should still work for actual vector
+      const testVector = new Array(32).fill(0.5);
+      await table.add([{ vector: testVector }]);
+
+      expect(await table.countRows()).toEqual(4);
```

Comment:
This test would be a lot better if it made an assertion about what the values in the `vector` column actually are at the end. See the test right below for an example of how to assert the values are equal to the expected ones.